doc/component/form-group.md

```diff
@@ -1,2 +1,54 @@
 # form-group
-[TBD]
+The `<form-group />` component allows you to render another a component and his label.
+
+## Attributes
+
+The following attributes (bindables) are available for this component.
+
+| attribute | type | default | description |
+|---|---|---|---|
+| behavior | string | `'input'` | The form behavior (horizontal, inline, etc) |
+| element | string | `'input'` | The component to render |
+| type | string | `'text'` | The type of component to render (text, radio, ect...)|
+| value | string | `undefined` | The value of the element |
+| name | string | `''` | The name of the element |
+| label  | string | `''` | The label of the element |
```

Comment:
Maybe add _(doesn't render label when empty)_

Maybe also add to element, message and error.